notebooks/tachyon/Simple Test1.snb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
Do you really want this test to be kept? If o maybe a bit of doc and renaming would be cool